gramps/plugins/db/dbapi/dbapi.py

```diff
@@ -78,10 +105,12 @@ class DBAPI(DbGeneric):
     Database backends class for DB-API 2.0 databases
     """
 
+    dbapi: Any  # might be better to have a specific ConnectionBase class
```

Comment:
```
    dbapi: Any  # might be better to have a specific ConnectionBase class
```
Flagging this line for comment.
It's not correct to type hint `dbapi` as `Any` but I could not find an appropriate superclass to use.